Clean up YCQL transactions page (#16667)
yugabyte/yugabyte-db: docs/content/preview/develop/learn/acid-transactions-ycql.md

- **Consistency** A completed transaction leaves the database in a consistent internal state. This can either be all the operations in the transactions succeeding or none of them succeeding.

- **Isolation** This property determines how/when changes made by one transaction become visible to the other. For example, a *serializable* isolation level guarantees that two concurrent transactions appear as if one executed after the other (that is, as if they occur in a completely isolated fashion). YugabyteDB supports *Snapshot* isolation level in the YCQL API. Read more about the different [levels of isolation](../../../architecture/transactions/isolation-levels/).

- **Durability** The results of the transaction are permanently stored in the system. The modifications must persist even in the instance of power loss or system failures.

{{<note title="Note">}}
Although YugabyteDB supports only *Snapshot* isolation level in the YCQL API, it supports three levels of isolation in the [YSQL](../../../explore/transactions/isolation-levels) API: *Snapshot*, *Serializable*, and *Read Committed*
{{</note>}}

## Transactions property

> The table should be created with the `transactions` property enabled. The statement should look something as follows.

Reviewer: why?

Author: Every time there is a mention of YCQL isolation in discussions, it is always mentioned that YCQL supports only Snapshot , but YSQL supports three ..

Reviewer: No, under Transactions property, I was wondering why "The table should be created with the `transactions` property enabled". There's no context. Are you saying that when creating tables in YCQL, the transactions property must be set to true if you want to have transactions? (why would you not?).

Author: Yes. To enable transactions in YCQL , the table should be created with the transactions property. Or else it wont work.

Reviewer: OK the transition seemed abrupt, suggestion below

Author: Yes. I noticed that too. Let's revisit the page & restructure it later.

> The table should be created with the `transactions` property enabled. The statement should look something as follows.

Reviewer: ```suggestion
To enable distributed transactions on tables in YCQL, create tables with the `transactions` property enabled, as follows:
```

```sql
CREATE TABLE IF NOT EXISTS <TABLE_NAME> (...) WITH transactions = { 'enabled' : true };
```

## Example with ycqlsh

> ##### Create keyspace and table

Reviewer: H3s should be fine here, no?

Author: To remove it from the ToC ..

Reviewer: Yes, thought it would be fine to have these headings in the rightnav.

Author: In my initial discussions with Karthik, he wanted to clean up the ToC on this page too.
